app/js/pluginManager.js

```diff
@@ -135,6 +129,16 @@ function PluginManager() {
 					event.args[1].left += $('#sidebar').width();
 					UI.tooltip.apply(null, event.args);
 					break;
+				case 'config':
+					// get or set something in the config.json
+					// TODO: Document in PluginTutorial.md
```

Comment:
I actually realized I had change the portion of the pluginManager where I detailed all the IPC channels and instead said to look at the code :P